examples/pythonDriver/hydaulicFractureWithMonitor/hydraulicFractureWithMonitor.py

```diff
@@ -0,0 +1,272 @@
+
```

Comment:
@rrsettgast - here is my working example.  It is currently setup to monitor the fracture extents and aperture every 10 cycles, and optionally push the time-history to a figure (on screen and/or to a png).  It's a bit long, since I've set it up to handle parallel runs.  A lot of this could be hidden behind a python module when ready.

I'm relying on the virtual python environment, which is built using `make geosx_xml_tools`.  In this PR, the source python is set to point to the version that @corbett5 has set up.  To run the example, you can do the following:

```bash
# Load the python environment
source ~/GEOS/geosx/GEOSX/build-quartz-gcc@8.1.0-release/python/start_geosx.sh

srun -n 32 -ppdebug python hydraulicFractureWithMonitor.py -i heterogeneousInSitu_singleFracture.xml -x 4 -y 2 -z 4
```

Reply:
Here's a sample of the output:

```
    Attempt:  0, NewtonIter:  3
    ( Rfluid, Rsolid ) = ( 6.63e-07, 7.64e-12 )
    ( R ) = ( 6.63e-07 ) ; 
    Last LinSolve(iter,res) = (   0, 0.00e+00 ) ; 
hydrofracture: Newton solver converged in less than 20 iterations, time-step required will be doubled.
Fracture Extents (m): min=[2.0000, 0.0000, -14.0000], max=[94.0000, 0.0000, 58.0000]
Aperture (mm): min=0.1013, max=2.2597
Resuming from step 7 of the event loop.
Time: 648s, dt:4s, Cycle: 170
Time: 652s, dt:4s, Cycle: 171
```

And here's an example of the figures being drawn on the fly:

![image](https://user-images.githubusercontent.com/4932756/92151223-9f9e7380-edd5-11ea-9736-297932cf6500.png)